src/structures/Invite.js

```diff
@@ -106,12 +106,24 @@ class Invite extends Base {
       this.maxUses ??= null;
     }
 
+    if ('inviter_id' in data) {
+      /**
+       * The user's id who created this invite
+       * @type {?Snowflake}
+       */
+      this.inviterId = data.inviter_id;
+      this.inviter = this.client.users.resolve(data.inviter_id);
```

Comment:
This should really be turned to a getter in the next major version, maybe slap a todo on this? CC @discordjs/the-big-4

Reply:
Sure 👍🏼

Reply:
Will that also be the case for `this.channel`?

Reply:
Yes

Reply:
Added with 78477b2fb28e5754e475ed25547b96acdac47d49!